src/llm_vm/onsite_llm.py

```diff
@@ -461,8 +465,8 @@ def generate(self,prompt, max_length=100,**kwargs): # both tokenizer and model t
         return ans['choices'][0]['message']['content']
 
     def finetune(self, dataset, optimizer, c_id):
-        print("fine tuning isn't supported by OpenAI on this model", file=sys.stderr)
-        exit()
+        message = "fine tuning isn't supported by OpenAI on this model"
```

Comment:
After #210 pr should be fixed this problem, but i don't have to an opportunity to test it, if you have an API key, please test and if it's working you can uncomment the below codes!
OpenAI allows the finetune with this specific model `gpt-3.5-turbo-0613` for more details please read the comments in this pr #210